### PR TITLE
fix: rename of `clasqaDB` repo to `clas12-qadb`

### PR DESCRIPTION
**Pull request by c-dilks**
Respects https://github.com/JeffersonLab/clasqaDB/issues/28

This is not really necessary, since old Github URLs always redirect, but still good to do.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -7,6 +7,6 @@
 [submodule "rcdb"]
 	path = rcdb
 	url = https://github.com/JeffersonLab/rcdb
-[submodule "clasqaDB"]
-	path = clasqaDB
-	url = https://github.com/JeffersonLab/clasqaDB
+[submodule "clas12-qadb"]
+	path = clas12-qadb
+	url = https://github.com/JeffersonLab/clas12-qadb
```
Comment (c-dilks) [inline .gitmodules:12]:
```suggestion
[submodule "clasqaDB"]
	path = clasqaDB
	url = https://github.com/JeffersonLab/clasqaDB
```

**Comment by c-dilks** [comment]
Github identifies submodule removal as a `git` conflict, so adding it back for now. We can remove it in a separate PR.

**Comment by c-dilks** [comment]
The same CI failures occur for both QADB `v1.1.1` and the latest version. Not sure what the issue is here.

**Comment by c-dilks** [comment]
@dglazier in principle this is ready for review, but I worry something is still broken, based in the CI failures.
- the current QADB submodule is out of date (tag v1.1.1); this PR updates the (newly-named) submodule to the latest version
- in both versions, the [`test` CI job fails](https://github.com/JeffersonLab/clas12root/actions/runs/5683993734/job/15405886686)
- the old `clasqaDB` submodule should be removed (`git rm clasqaDB`), but pushing such a commit to this PR causes Github to flag it as a conflict, which blocks the CI and merging; so this should be done separately (maybe a commit directly to the main branch)

**Comment by c-dilks** [comment]
Sorry, I just realized my fork was out of date. Rebased and trying again.

**Comment by dglazier** [comment]
Thanks Chris (sorry for delay I was on vacation). It seems to be a rogue test. It is failing on the build, which has already successfully passed in the previous test. The tests were written by a student who has since moved on and should either be updated or removed. I downloaded your code and it built fine, so I am happy to  merge the pull request if you think it is ready.

**Comment by c-dilks** [comment]
> It seems to be a rogue test. It is failing on the build, which has already successfully passed in the previous test.

#55 should fix that. Otherwise this PR is ready.

**Pull request merged**